pkg/order/mempool/utils.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/meshplus/bitxhub-model/pb"
 	"github.com/meshplus/bitxhub/internal/constant"
 	raftproto "github.com/meshplus/bitxhub/pkg/order/etcdraft/proto"
+
```

Comment:
go fmt?

Reply:
left a blank between the external and internal package.